### PR TITLE
week 4 assignment

### PR DESCRIPTION
**Pull request by sachin286**

### CODE REVIEW

File: Week-4/Exercise-4.2/inheritance.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var Person = function () {};
```
Comment (ShashankSuresh) [inline Week-4/Exercise-4.2/inheritance.js]: The `var `keyword is considered dead in JS world. So lets use either `let `or `const`. In this scenario, lets use `const`.

File: Week-4/Exercise-4.2/inheritance.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    console.log(this.name + "is of age " + this.age);
+  };
+};
+
+Teacher.prototype = new Person();
+
+var him = new Teacher();
+
+him.initialize("Adam", 45);
+him.teach("Inheritance");
```
Comment (ShashankSuresh) [inline Week-4/Exercise-4.2/inheritance.js]: Per our discussion, kindly make the changes :)

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

**Comment by sachin286** [comment]
requested changes updated

**Review by ShashankSuresh — APPROVED** [review]
Looks good. Good work. :)

**Pull request merged**